src/Mold.cpp

```diff
@@ -0,0 +1,31 @@
+/** Copyright (C) 2017 Ultimaker - Released under terms of the AGPLv3 License */
+#include "Mold.h"
+#include "utils/intpoint.h"
+
+namespace cura
+{
+
+void Mold::process(Slicer& slicer, coord_t layer_height, double angle, coord_t width, coord_t open_polyline_width)
+{
+    Polygons mold_outline_above; // the outside of the mold
+
+    coord_t inset = tan(angle / 180 * M_PI) * layer_height;
```

Comment:
I'm getting a really large number, rather than NaN. M_PI is not exactly pi of course...
I don't think this can fail.
Maybe I should do a `std::max(0, ...)` though